lib/loadIcons.js

```diff
@@ -30,4 +30,13 @@ export const icon = {
   'school': solid('school'),
   'stamp': solid('stamp'),
   'user-check': solid('user-check'),
+  'money-bill-transfer': solid('money-bill-transfer'),
+  'phone-flip': solid('phone-flip'),
+  'location-dot': solid('location-dot'),
+  'circle-minus': solid('circle-minus'),
+  'circle-plus': solid('circle-plus'),
+  'address-book': solid('address-book'),
+  'map-location-dot': solid('map-location-dot'),
+  'comment-dots': solid('comment-dots'),
+  'bell-solid 1': solid('bell'),
```

Comment:
It is worth putting these in alphabetical order so we can quickly see if icons are already added

Reply:
ordered icons